docs/cli.rst

```diff
@@ -1,114 +1,222 @@
 Command Line Tool
 =================
 
-NAPALM ships with a simple CLI tool to help you deploying configuration to your devices directly from the shell.
-It might be convenient for simple bash scripts or provisioning tools that rely on a shell.
+NAPALM ships with a very simple cli tool so you can use napalm straight from the CLI. It's use is quite simple and you can see the help with ``--help``:
 
-The usage is very simple. For example, let's do a dry run (changes will not be applied) and check the changes between
-my current configuration and a new candidate configuration:
 
-.. code-block:: diff
+.. code-block::
```

Comment:
I would just add `.. code-block:: bash`, but there's no much visual rendering gain, so it's okay like that.